.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: ci
+on: [push, pull_request]
```

Comment:
to avoid too many runs and since master is protected may be we just need to have `pull_request` (i never use `push` on projects)

Reply:
are just keep `push` but currently it seems that ci is ran twice

Reply:
done